.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
```

Comment:
We probably need to specify `cache: "npm"` action argument to ensure npm dependencies are cached for faster build times.

Reply:
That file was autogenerated and I do not care for now for running it as part of the CI/CD pipeline. My top priority is to fix the Python code, so that I am not blocking @zdaratom when he returns.

I am skipping this for now.